app/parse.py

```diff
@@ -15,8 +34,103 @@ class Product:
     num_of_reviews: int
 
 
+class ChromeDriver:
+    def __init__(self) -> None:
+        self._driver = webdriver.Chrome()
+
+    def __enter__(self) -> webdriver.Chrome:
+        return self._driver
+
+    def __exit__(
+            self,
+            exc_type: Optional[Type[BaseException]],
+            exc_val: Optional[BaseException],
+            exc_tb: Optional[TracebackType]
+    ) -> None:
+        self._driver.close()
```

Comment:
It's a good practice to use `quit()` instead of `close()` to ensure that all browser windows are closed and the WebDriver process is terminated.